BraveWallet/NetworkIcon.swift

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import BraveCore
+import SwiftUI
+
+struct NetworkIcon: View {
+  
+  var network: BraveWallet.NetworkInfo
+
+  @ScaledMetric private var length: CGFloat = 30
+  
+  var body: some View {
+    Group {
+      if network.chainId != BraveWallet.PolygonMainnetChainId,
+          let (iconName, grayscale) = networkImageInfo {
+        Image(iconName, bundle: .current)
+          .resizable()
+          .aspectRatio(contentMode: .fit)
+          .grayscale(grayscale ? 1 : 0)
+          .clipShape(Circle()) // don't clip polygon
```

Comment:
I would prefer to keep the asset images as-is. Presumably you are talking about adding padding to the asset image file itself so it doesn't get clipped by the circle, but if we later got a larger dimension asset file it would then need edited to add padding too. I think I'd prefer to keep this style, and it's contained into `NetworkIcon` so should be okay.

Reply:
Fixed in [d42e960](https://github.com/brave/brave-ios/pull/5706/commits/d42e9609d512ed4437b8b0cd10e8ed06ba9368c7). Using pdfs for the svg assets, and no more clipping in `NetworkIcon`. Updated Solana asset to use the rounded icon.